hometask1_1-java_14-features/src/main/java/ui/pages/components/PageFooter.java

```diff
@@ -0,0 +1,14 @@
+package ui.pages.components;
+
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.ElementsCollection;
+
+public class PageFooter {
+    public ElementsCollection getSocialLinks(){
+        $(".social").should(Condition.visible);
```

Comment:
Avoid this hardcoding please. It isn't clear, what this element is.

Create variable and then reuse it here. For example:
private SelenideElement 'nameOfElement' = .......

And for the lower element the same